invenio_files_rest/errors.py

```diff
@@ -41,3 +41,11 @@ class FileInstanceAlreadySetError(FilesException):
 
 class InvalidOperationError(FilesException):
     """Exception raise when an invalid operation is performed."""
+
+
+class MultipartObjectException(Exception):
```

Comment:
Subclass from ``FilesException`` to ensure that all exceptions in the class have a common base class?